clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/Task.groovy

```diff
@@ -82,4 +94,17 @@ public interface Task {
    * This returns the start time of the Task's execution in milliseconds since epoch form.
    */
   long getStartTimeMs()
+
+  /**
+   * Connect a Saga to this Task. More than one Saga can be associated with a Task.
+   *
+   * @param sagaId The Saga name/id pair
+   */
+  void connectSaga(@Nonnull SagaId sagaId);
+
+  /**
+   * A list of Sagas associated with this Task, if any.
+   */
+  @Nonnull
+  List<SagaId> getSagaIds();
```

Comment:
nit: maybe
```suggestion
  List<SagaId> getConnectedSagas();
```
to be symmetrical with `connectSaga`?

Reply:
I went the other way and just got rid of the `connect` taxonomy. It was a silly thing.